.github/workflows/integration.yml

```diff
@@ -0,0 +1,65 @@
+name: Integration test
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - { python-version: '3.12', region: us-west-1, access-key-id: '000000000001'}
+          - { python-version: '3.12', region: us-east-1, access-key-id: 'test'}
+          - { python-version: '3.11', region: us-east-1, access-key-id: 'test'}
+          - { python-version: '3.10', region: us-east-1, access-key-id: 'test'}
+          - { python-version: '3.9', region: us-east-1, access-key-id: 'test'}
+          - { python-version: '3.8', region: us-east-1, access-key-id: 'test'}
+
+    env:
+      AWS_ACCESS_KEY_ID: ${{ matrix.access-key-id }}
+      AWS_REGION: ${{ matrix.region }}
+      AWS_DEFAULT_REGION: ${{ matrix.region }}
+      AWS_SECRET_ACCESS_KEY: test
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: '${{ matrix.python-version }}'
+
+      - name: Install localstack CLI
+        run: pip install localstack
+
+      - name: Install deps
+        run: pip install '.[ver1]'
+
+      - name: Check aws-cli version
+        run: |
+          aws --version
+          awslocal --version
+          which aws
+          which awslocal
+
+      - name: Start and wait for localstack (Community)
```

Comment:
nit: At some point in the future we could start dogfooding our own Github Action.. https://github.com/localstack/setup-localstack 😎 (no need to change for now, though..)